### PR TITLE
Describe how filter query logic and types can be reused

### PR DESCRIPTION
**Pull request by KnorpelSenf**
As discussed in https://t.me/grammyjs/48027

### CODE REVIEW

**Comment by Satont** [comment]
Can `matchFilter ` and `Filter` be splitted as different sections? So It will be easier to link to each one. I saw so many questions in grammY chat about how to do `Filter`.

**Comment by Satont** [comment]
LGTM 👍🏻

**Review by Satont — APPROVED** [review]

**Review by roj1512 — APPROVED** [review]

**Comment by KnorpelSenf** [comment]
Why force push? What happened here?

**Comment by roj1512** [comment]
Hmm, I didn't do anything around this, to be honest.

**Comment by roj1512** [comment]
Perhaps it is a bug.

**Comment by KnorpelSenf** [comment]
Oh I see, it was the target branch `main` which was force-pushed

**Comment by roj1512** [comment]
Yes, I did it on `main` and `add-deno-bin`. They are going to be my last force pushes anyway.

**Comment by KnorpelSenf** [comment]
I appreciate that :D

File: site/docs/guide/filter-queries.md [hunk 1]
```diff
@@ -319,3 +354,6 @@ These functions are then combined to form a predicate that only has to check for
 
 This system uses less operations than some competing libraries, which need to perform containment checks in arrays when routing updates.
 grammY's filter query system is much more powerful.
+
+```
```
Comment (WingLim) [inline site/docs/guide/filter-queries.md:358]: Why we have this without any code in it?

File: site/docs/guide/filter-queries.md [hunk 2]
```diff
@@ -277,6 +277,41 @@ bot.on("message").filter(
...
+
+You can import `matchFilter` directly if you want to use it in your own logic.
+For example, you can decide to drop all updates that match a certain query:
+
+```ts
+// Drop all text messages or text channel posts.
+bot.drop(matchFilter(":text"));
+```
+
+Analogously, you can make use of the filter query types that grammY uses internally:
```
Comment (WingLim) [inline site/docs/guide/filter-queries.md:294]:
```suggestion
Analogously, you can make use of the filter query types that grammY uses internally.
```
Should it be a `.` not `:` ?

File: site/docs/guide/filter-queries.md (hunk 1 above)
Comment (roj1512) [reply site/docs/guide/filter-queries.md]: Fixed.

Comment (KnorpelSenf) [reply site/docs/guide/filter-queries.md]: That's an artefact of formatting Markdown with incomplete code environments. The two lines should be removed.

**Review by roj1512 — APPROVED** [review]

**Pull request merged**